spec/javascript/components/molecules/StatusItems.test.js

```diff
@@ -1,43 +1,139 @@
 import React from 'react'
-import renderer from 'react-test-renderer'
-import { mockStatusItem, mockStatusItems } from '../../mocks/statusItemMock'
+import { shallow } from 'enzyme'
+import { mockManyStatusItems, mockStatusItem, mockStatusItems } from '../../mocks/statusItemMock'
 import StatusItems from 'components/molecules/StatusItems'
+import StatusItem from 'components/molecules/StatusItem'
 
-const getWrapper = (items, limit = null) => renderer.create(
+const getWrapper = (items, limit = undefined) => shallow(
   <StatusItems
     statusItems={items}
     limit={limit}
   />
 )
 
+const TIMESTAMP_AUGUST_25_2020 = 1598338800
+const TIMESTAMP_AUGUST_26_2020 = 1598400000
+
 describe('StatusItems', () => {
-  test('should render empty status items correctly', () => {
-    const wrapper = getWrapper([])
-    expect(wrapper).toMatchSnapshot()
+  describe('snapshot tests', () => {
```

Comment:
Snapshot tests can still sometimes be useful, but our main tests should be specific unit tests. This file has some snapshot and some unit tests